### ISSUE TITLE
Is there a way to use this plugin within my API controllers?

### ISSUE THREAD

**Issue by selected-pixel-jameson** [ComfortablyCoding/strapi-plugin-transformer#43]
Is there a way to use this plugin within my API controllers? For example I'm hooking into a process when a model is updated and I want to send a request to a third-party service with this data.

**Comment by ComfortablyCoding** [comment]
Yes the transform functionality is just a service of the plugin so it can be used anywhere you have access to the strapi obj.
